people-and-planet-ai/geospatial-classification/requirements-test.txt

```diff
@@ -0,0 +1,5 @@
+earthengine-api==0.1.292
```

Comment:
Any requirement that is already on the requirements.txt file is not necessary here. This is like an "additional" requirements for testing. Nox will install requirements.txt first, and then install requirements-test.txt on top of it. So I think we can keep only `pytest` and `pytest-xdist` :)